devlab/tests/test_resource_migration.py

```diff
@@ -9,9 +9,25 @@
 from fabric.api import run, settings
 from fabric.network import NetworkError
 
+NET_NAMES_TO_OMIT = ['tenantnet4_segm_id_cidr1',
+                     'tenantnet4_segm_id_cidr2']
+SUBNET_NAMES_TO_OMIT = ['segm_id_test_subnet_1',
+                        'segm_id_test_subnet_2']
+PARAMS_NAMES_TO_OMIT = ['cidr', 'gateway_ip', 'provider:segmentation_id']
+
 
 class ResourceMigrationTests(functional_test.FunctionalTest):
 
+    def _is_segm_id_test(self, param, name):
+        result = False
+        if param in PARAMS_NAMES_TO_OMIT\
```

Comment:
pls re-write like this:
if param in PARAMS_NAMES_TO_OMIT and ( name in PARAMS_NAMES_TO_OMIT or name in SUBNET_NAMES_TO_OMIT)